kivy/graphics/context.pyx

```diff
@@ -25,11 +25,17 @@ from kivy.graphics.texture cimport Texture, TextureRegion
 from kivy.graphics.vbo cimport VBO, VertexBatch
 from kivy.logger import Logger
 from kivy.clock import Clock
-from kivy.graphics.c_opengl cimport *
-IF USE_OPENGL_MOCK == 1:
-    from kivy.graphics.c_opengl_mock cimport *
-IF USE_OPENGL_DEBUG == 1:
-    from kivy.graphics.c_opengl_debug cimport *
+
+from kivy.graphics.c_opengl_def cimport *
+IF USE_OPENGL_DEBUG:
+    cimport kivy.graphics.c_opengl_debug as cgl
+ELIF USE_OPENGL_DYNAMIC:
+    from kivy.graphics.c_opengl_dynamic cimport cgl
+ELIF USE_OPENGL_MOCK:
+    cimport kivy.graphics.c_opengl_mock as cgl
+ELSE:
+    cimport kivy.graphics.c_opengl as cgl
+
```

Comment:
Somethine like KIVY_GL_BACKEND, set to either ANGLE, DYNAMIC, MOCK, DYNAMIC_DEBUG, STATIC, STATIC_DEBUG, etc